test/e2e/pod_create_test.go

```diff
@@ -1213,4 +1214,131 @@ ENTRYPOINT ["sleep","99999"]
 		Expect(strings[0]).Should(ContainSubstring("size=10240k"))
 	})
 
+	FIt("podman pod create --restart", func() {
```

Comment:
Consider turning this into a table test.  It's very large and hence hard to read/maintain.

Reply:
Cleaned up the tests a bit and split them up.